Rokt-Widget.podspec

```
@@ -0,0 +1,32 @@
+#
+# Be sure to run `pod lib lint rokt.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'Rokt-Widget'
+  s.version          = '4.9.0'
+  s.summary          = 'Rokt Mobile SDK to integrate ROKT Api into iOS application'
+  s.swift_version    = '5.0'
```

```suggestion
  s.swift_version    = '6.0'
```

Given we're changing to Xcode 16 the default swift version would now be 6.0 I believe

---

Updated